lib/arbre/html/tag.rb

```diff
@@ -115,25 +116,18 @@ def closing_tag
       def indent(open_tag, child_content, close_tag)
         spaces = ' ' * indent_level * INDENT_SIZE
 
-        html = ""
-
-        if no_child? || child_is_text?
+        spaces + if no_child? || child_is_text?
```

Comment:
Can we keep the `html` local and use `+=`? This inline `+ if` style looks very weird.